app/controllers/catalog_controller.rb

```diff
@@ -71,14 +71,16 @@ class CatalogController < ApplicationController
     # ------------------------------------------------------------------------------------------
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
-    config.add_index_field solr_name("repository",        :stored_sortable), label: "Library", helper_method: :render_repository_facet_link
-    config.add_index_field solr_name("unittitle",         :displayable),  :label => "Title", :highlight => true, :helper_method => :render_field_item
-    config.add_index_field solr_name("abstract",          :displayable),  :label => "Abstract", :highlight => true, :helper_method => :render_field_item
+    #config.add_index_field solr_name("unittitle",         :displayable),  :label => "Title", :highlight => true, :helper_method => :render_field_item
     config.add_index_field solr_name("format",            :displayable),  :label => "Format", :helper_method => :render_field_item
-    config.add_index_field solr_name("language",          :displayable),  :label => "Language", :helper_method => :render_field_item
-    config.add_index_field solr_name("unitdate",          :displayable),  :label => "Dates", :helper_method => :render_field_item
-    config.add_index_field solr_name("collection",        :displayable),  :label => "Archival Collection", :helper_method => :render_collection_facet_link, :highlight => true
-    config.add_index_field solr_name("parent_unittitles", :displayable),  :label => "Series", :highlight => true, :helper_method => :render_series_facet_link
```

Comment:
@eshadatta I'm confused about this. It looks like you duplicated existing functionality. The `parent_unittitles` is indexed as an array of parent unittitles and the `render_series_facet_link` creates faceted links to each of the parents.

It appears that the `render_breadcrumb` function does exactly what `render_series_facet_link` does. Unless there is a reason for this I'm missing I think you can just change the existing code

```
config.add_index_field solr_name("parent_unittitles", :displayable),  :label => "Series", :highlight => true, :helper_method => :render_series_facet_link
```

To

```
config.add_index_field solr_name("parent_unittitles", :displayable),  :label => "Contained in", :highlight => true, :helper_method => :render_series_facet_link
```

It just becomes a matter of relabelling and you can remove all the changes in the index end so we don't have to reindex again.